dts/arm/st/h7/stm32h747.dtsi

```diff
@@ -5,3 +5,21 @@
  */
 
 #include <st/h7/stm32h745.dtsi>
+#include <zephyr/dt-bindings/mipi_dsi/mipi_dsi.h>
+
+/ {
+	soc {
+		mipi_dsi: dsihost@50000000 {
+			compatible = "st,stm32-mipi-dsi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x50000000 0x1000>;
+			clock-names = "dsiclk", "refclk", "pixelclk";
+			clocks = <&rcc STM32_CLOCK_BUS_APB3 0x00000010>,
+				 <&rcc STM32_SRC_HSE DSI_SEL(1)>,
+				 <&rcc STM32_SRC_PLL3_R DSI_SEL(1)>;
```

Comment:
From what I see, you're only using those to get reference clocks freqs, and there is no way to configure DSI domain clock (for now at least).
This is somehow confusing as you're providing 'DSI_SEL(1)' which lets suppose that you're actually configuring a domain clock. In order to avoid confusion, I'd suggest to use `NO_SEL` instead.
From some reason `NO_SEL` value is not available on H7 series so you'd need to add it (cf definition in other series).

Also, adding some notes about it in bindings would be valuable to make it even clearer.